.github/workflows/tox-test.yml

```diff
@@ -100,6 +100,65 @@ jobs:
         uses: pypa/gh-action-pip-audit@v1.1.0
         with:
           inputs: requirements.txt requirements-test.txt
+      - name: Cache OWASP
+        uses: actions/cache@v3
```

Comment:
can we bump version similar as in cloudpub?

Reply:
Oh, yes!

Reply:
Updated